client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -461,8 +473,17 @@ const mapSelectToProps = withSelect( ( select, props ) => {
 
 	const { paymentId } = props.attributes;
 
+	const fields = [
+		'content',
+		'meta.spay_currency',
+		'meta.spay_email',
+		'meta.spay_multiple',
+		'meta.spay_price',
+		'title.raw',
+	];
```

Comment:
should check if:
```

	const fields = [
		'content',
		[ 'meta', 'spay_currency' ],
        // ...
	];
```
...works.

Reply:
That would work. Changing to array-paths was a bit stylistic, but:

`'content' === [ 'content' ]`
`'meta.spay_currency' === [ 'meta', 'spay_currency' ]`

Lodash will parse the strings into parts (even if it ends up with just one part, it doesn't know that initially), but the right hand array versions will be used directly.